queries.sql

```diff
@@ -23,3 +23,30 @@ SELECT * FROM animals WHERE name NOT IN ("Gabumon");
 
 -- 8.0 Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+-- Part 2 Questions
+
+-- 1. How many animals are there?
```

Comment:
- [ ] It would be better if we UPDATE the animals table by setting the species column to unspecified, then roll back the change (**for this step students are explicitly asked to verify that change was made and rollbacked, so make sure that they included all steps as in the example below **). However, I did not see this

Reply:
@addod19 , the queries.sql only contains the queries for the questions as per the given instructions. Am i supposed to include the  transactions too? or just screenshots?